docs/inputs.md

```diff
@@ -1,4 +1,13 @@
 
+# `autocomplete`
+UI that reveals potential matches to an entered value, ie a value entered via `simple-list`.
+
+### Autocomplete Arguments
+* **list** - name of a list that Amphora knows about accessible via `/<site>/_lists/<listName>`.
+* **allowDestroy** - boolean that determines whether the user can delete items from the list (so it doesn't appear as a match in the autocomplete). THIS IS SET VIA THE PARENT OF AUTOCOMPLETE.
```

Comment:
I don't really understand this line. Is this specified in a schema? If not, I'd remove the autocomplete docs and leave the `modifyList` information in the relevant inputs, so as to not confuse devs